<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StorageSettingsWidget</class>
 <widget class="QWidget" name="StorageSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paths</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Storage root path:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditStorageRoot"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonStorageRoot">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="folder">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Exercises root path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditExercisesRoot"/>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButtonExercisesRoot">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="folder">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="gbSchemeFilepaths">
     <property name="title">
      <string>Scheme filepaths</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="lSchemeFilepaths"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
